include/shellstrings.h

```diff
@@ -89,31 +122,80 @@ void shell_input(char* input_file, char* status);
  * Returns:
  *  - None.
  */
-void shell_help();
+void shell_help(bool *color);
 
 /*
- * shell_usage - command line usage instructions
+ * shell_start_interactive - prints out interactive starting message
  *
  * Parameters:
+ *  - file, dictionary, mode names
+ *
+ * Returns:
  *  - None.
+ */
+void shell_start_interactive(char *filename, char *dict, char *md, bool *color);
+
+
+/*
+ * shell_modename - prints out mode name (verbose, quiet or interactive)
+ * 
+ * Accepts flag int
+ *      note 1 is quiet, 2 is batch, 3 is interactive
```

Comment:
Please `#define` your own verbosities and use them, rather than magic numbers like these.

Reply:
Please make this header consistent with the others in terms of formatting.

Reply:
updated!